components/desktop/awesome/Makefile

```diff
@@ -60,3 +61,5 @@ REQUIRED_PACKAGES += x11/library/xcb-util-cursor
 REQUIRED_PACKAGES += x11/library/xcb-util-keysyms
 REQUIRED_PACKAGES += x11/library/xcb-util-wm
```

Comment:
Please mark the auto-generated (gmake REQUIRED_PACKAGES) requirements with a line "# Auto-generated dependencies".
If you have added new dependencies that haven't been found by gmake REQUIRED_PACKAGES you should separate them an mark them accordingly.